contracts/Neumark.sol

```diff
@@ -68,6 +74,7 @@ contract Neumark is
     function burnNeumark(uint256 neumarkUlps)
         public
         only(ROLE_NEUMARK_BURNER)
+        acceptAgreement(msg.sender)
```

Comment:
please keep in mind that currently it is called by LockedAccount contract only

Reply:
I'm aware. This might change in the future, so we need to have it in. It might be redundant with the `mOnTransfer` one though. I'll check that.